README.md

```diff
@@ -1,2 +1,41 @@
-# proposals
-This repository holds all the past and current proposals for the Mixxx Software
+# Proposals
+
+This repository holds all the past and current proposals for the Mixxx Software. It’s the single place for
+reviewing, discovering, and working on the design documents. It is also a record of past decisions and approvals.
+
+## Current Proposals
+
+* The [open PRs with the `proposal` label](https://github.com/mixxxdj/proposals/pulls?q=is%3Aopen+is%3Apr+label%3Aproposal) show all the pending proposals.
+* The [proposals directory](./proposals) shows all the accepted proposals. See the “Implementation Status” for details on the implementation.
```

Comment:
Some thought, for adjusting the workflow a bit. 

The workflow proposed here, does not allow to "publish" a draft and collaborate before it is ready. You will have a proposal owner, which may accept suggestions until the proposal is "accepted."  This makes it hard when you for instance have a good idea, but no interest or time to actually implement it. 

After merge, an Owner without commit rights is no longer "Owner" other may hijack the proposal. This will not happen if the proposal is developed in the Owners repository and is only reflected as a PR in this repository. 
 
Currently we don't have a workflow for "accepting" a proposal. It is implicit accepted once you find a contributor and a reviewer for it. Both may collaborate on a proposal. How can we model this workflow?  

I think a proposal should be also a place to sort thoughts. 

In the past we had design discussions along PRs which in turn required extra work. My hope is that a proposal avoids it.
This requires that objections a phrased early. How can we archive this? 

Any ideas how we can reflect these topics as well?

Reply:
the PR process is how proposals are discussed and accepted.  When they are merged, the proposals are accepted

Reply:
I think the idea of the Owner here is someone who cares about driving the effort to make the proposal happen. It's isn't necessary the initial contributor (e.g cases where the original idea gets abandonned and re-openned by someone else, or case where one ask for a feature but doesn't have the time, knowledge or skills to make the idea happened).

This owner will help with:
- informing on changes on the initial plans, in case things gets adjusted after accepting a proposal
- facilitating contacts (e.g on Zulip)
- crediting the original effort on driving the change.

The advantage of writing down the owner(s) in the document is to prevent cases where the commit authors isn't the original author (as you suggested with a "hijack": this would likely lead to second owner being added, working in collaboration with the original owner, or replacing in case the former owners have lost interest in driving the feature.

Finally, "accepting" doesn't mean having the feature ready and implmented in Mixxx, but rather agreeing on making the proposal happen as time goes. For example, if the proposal repo was here at that time, the QML proposal would have already being accepted (merged) although the development effort is still ongoing. This is a way to say "Mixxx is committed to modernising the UI using QML". Does that make sense? Do you think further info  should go in the README.md?

Reply:
@daschuer do you understand the process now or is the wording still confusing?